code/modules/mob/new_player/new_player.dm

```diff
@@ -309,6 +309,7 @@
 		if(prob(10)) // 10% of those have a good mut.
 			H.dna.GiveRandomSE(notflags = GENE_UNNATURAL,genetype = GENETYPE_GOOD)
 
+var/global/list/meteor_spawn = list("Trader")
```

Comment:
I don't really like this: perhaps you can make `meteor_spawn` a boolean on `/datum/job`, then use `GetJob` to check for it?

Reply:
That's what I did originally until I realized that the spawning proc doesn't actually reference real rank at all, just a rank var that's a string. Kind of annoying.

Reply:
It looks like it should work? https://github.com/vgstation-coders/vgstation13/blob/7297652aa440a8c70be3317ff2facd9897b8dca1/code/game/jobs/job_controller.dm#L54

Reply:
AttemptLateSpawn(rank) is a mob proc not a job datum proc

Reply:
That's like the least clear way to say "there exists a GetJob(rank) proc"!